api/routes/index.js

```diff
@@ -0,0 +1,432 @@
+  validate(measurement, 'participantAddress', { type: 'ethereum address', required: true })
+  validate(measurement, 'timeout', { type: 'boolean', required: false })
+  validate(measurement, 'startAt', { type: 'date', required: false })
+  validate(measurement, 'statusCode', { type: 'number', required: false })
+  validate(measurement, 'headStatusCode', { type: 'number', required: false })
+  validate(measurement, 'firstByteAt', { type: 'date', required: false })
+  validate(measurement, 'endAt', { type: 'date', required: false })
+  validate(measurement, 'byteLength', { type: 'number', required: false })
+  validate(measurement, 'attestation', { type: 'string', required: false })
+  validate(measurement, 'carTooLarge', { type: 'boolean', required: false })
+  validate(measurement, 'carChecksum', { type: 'string', required: false })
+  validate(measurement, 'indexerResult', { type: 'string', required: false })
+  validate(measurement, 'minerId', { type: 'string', required: false })
+  validate(measurement, 'providerId', { type: 'string', required: false })
+  validate(measurement, 'stationId', { type: 'string', required: true })
+  assert(measurement.stationId.match(/^[0-9a-fA-F]{88}$/), 400, 'Invalid Station ID')
+
+  const inetGroup = await mapRequestToInetGroup(request.pgPool, request.raw)
+  logNetworkInfo(request.headers, measurement.stationId, recordNetworkInfoTelemetry)
+
+  const { rows } = await request.pgPool.query(`
+      INSERT INTO measurements (
+        spark_version,
+        zinnia_version,
+        cid,
+        provider_address,
+        protocol,
+        participant_address,
+        station_id,
+        timeout,
+        start_at,
+        status_code,
+        head_status_code,
+        first_byte_at,
+        end_at,
+        byte_length,
+        attestation,
+        inet_group,
+        car_too_large,
+        car_checksum,
+        indexer_result,
+        miner_id,
+        provider_id,
+        completed_at_round
+      )
+      SELECT
+        $1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14, $15, $16, $17, $18, $19, $20, $21,
+        id as completed_at_round
+      FROM spark_rounds
+      ORDER BY id DESC
+      LIMIT 1
+      RETURNING id
+    `, [
+    measurement.sparkVersion,
+    measurement.zinniaVersion,
+    measurement.cid,
+    measurement.providerAddress,
+    measurement.protocol,
+    measurement.participantAddress,
+    measurement.stationId,
+    measurement.timeout || false,
+    parseOptionalDate(measurement.startAt),
+    measurement.statusCode,
+    measurement.headStatusCode,
+    parseOptionalDate(measurement.firstByteAt),
+    parseOptionalDate(measurement.endAt),
+    measurement.byteLength,
+    measurement.attestation,
+    inetGroup,
+    measurement.carTooLarge ?? false,
+    measurement.carChecksum,
+    measurement.indexerResult,
+    measurement.minerId,
+    measurement.providerId
+  ])
+  
+  return { id: rows[0].id }
```

Comment:
```suggestion
  reply.send({ id: rows[0].id })
```

Reply:
I suspect the other routes have similar issues. Once CI is fixed, that should show.